src/v2/Apps/Show/__tests__/ShowArtworks.jest.tsx

```diff
@@ -48,4 +48,116 @@ describe("ShowArtworks", () => {
     expect(wrapper.find("ArtworkFilterArtworkGrid").length).toBe(1)
     expect(wrapper.find("ArtworkGridItem").length).toBe(1)
   })
+
+  it("renders filters in correct order", () => {
+    const wrapper = getWrapper({
+      FilterArtworksConnection: () => ({
+        counts: {
+          followedArtists: 10,
+        },
+      }),
+    })
+    const filterWrappers = wrapper.find("FilterExpandable")
+    const filters = [
+      {
+        label: "Artists",
+        expanded: true,
+      },
+      {
+        label: "Rarity",
+        expanded: true,
+      },
+      {
+        label: "Medium",
+        expanded: true,
+      },
+      {
+        label: "Price",
+        expanded: true,
+      },
+      {
+        label: "Size",
+        expanded: true,
+      },
+      {
+        label: "Ways to buy",
+        expanded: true,
+      },
+      {
+        label: "Material",
+      },
+      {
+        label: "Artist nationality or ethnicity",
+      },
+      {
+        label: "Time period",
+      },
+      {
+        label: "Color",
+      },
+      {
+        label: "Galleries and institutions",
+      },
```

Comment:
We actually don't want this one according to [the ticket](https://artsyproduct.atlassian.net/browse/FX-3060), which also makes sense since a show is a single-partner page.

Reply:
Yes, you're right. I missed this moment

Reply:
My mistake is that in tests I pass mocked `PARTNER` aggregation, which should not be here